src/routes/login/+page.svelte

```diff
@@ -29,48 +122,60 @@
             />
+            {#if emailIndicator == indicatorStatus.warning}
+                <div
+                    class="flex flex-col justify-center items-center transition-opacity"
+                >
+                    <p class="text-sm text-warning-500">
+                        Bitte gib eine valide Email-Adresse ein
+                    </p>
+                </div>
+            {/if}
             <!-- Password field -->
             <label class="label" for="password_input">
                 <span>Passwort</span>
             </label>
-            <div
-                class="input-group input-group-divider relative grid-cols-[auto_1fr_auto]"
-            >
+            <div class="relative">
                 <input
-                    class="w-72 {passwordIndicator}"
+                    class="bg-surface-700 text-white border rounded-full transition-colors {passwordIndicator} w-full focus:border-primary-500 focus:outline-none focus:ring-0"
                     type={showPassword ? "text" : "password"}
                     id="password_input"
                     placeholder="Dein super sicheres Passwort 😉"
+                    bind:this={passwordReference}
                 />
-                <button
-                    class="variant-filled-secondary right-0 leading-5"
-                    on:click={() => (showPassword = !showPassword)}
-                >
-                    {#if showPassword}
-                        {btnIcon.unlocked}
-                    {:else}
-                        {btnIcon.locked}
-                    {/if}
-                </button>
+                <div>
+                    <button
+                        class=" absolute bg-surface-700 w-14 border-surface-500 inset-y-0 h-8 top-1 right-1 border-l-2 border-surface-400 pl-4 flex items-center text-md rounded-r-full"
+                        on:click={() => (showPassword = !showPassword)}
+                    >
+                        {#if showPassword}
+                            {btnIcon.unlocked}
+                        {:else}
+                            {btnIcon.locked}
+                        {/if}
+                    </button>
+                </div>
             </div>
 
             {#if somethingWrong}
                 <div class="flex flex-col items-center">
-                    <p class="text-sm text-error-500">
+                    <p class="text-sm text-error-500 transition-opacity">
                         Email oder Passwort sind falsch
                     </p>
                 </div>
             {/if}
 
             <!-- Login button -->
             <div class="flex flex-col items-center">
-                <button class="variant-filled-primary btn w-full"
-                    >Anmelden</button
+                <button
+                    class="btn variant-filled-primary w-full"
+                    on:click={tryLogin}>Anmelden</button
                 >
             </div>
             <!-- Password reset -->
```

Comment:
Kann das unten nicht markieren, weil das nicht in diesem PR geändert wurde. Aber kannst du maybe die Farben von "Passwort vergessen" und "Neuen Account erstllen" angleichen. Die unterscheiden sich ja schon in der Größe, das sollte reichen.
`text-tertiary-500` und `text-tertiary-600`

Reply:
fixed